modules/apps/commerce/commerce-frontend-taglib/src/main/resources/META-INF/resources/header/page.jsp

```diff
@@ -103,11 +103,11 @@ String myWorkflowTasksPortletNamespace = PortalUtil.getPortletNamespace(PortletK
 										<liferay-ui:icon-help message="external-reference-code" />
 
 										<c:if test="<%= Validator.isNotNull(externalReferenceCodeEditUrl) %>">
-											<clay:link
-												elementClasses="btn btn-link btn-unstyled header-info-link ml-3"
-												href="#"
+											<clay:button
+												displayType="unstyled"
+												icon="pencil"
 												id="erc-edit-modal-opener"
-												label='<%= LanguageUtil.get(request, "edit") %>'
+												small="<%= true %>"
 											/>
 
 											<aui:script require="commerce-frontend-js/utilities/eventsDefinitions as events">
```

Comment:
Once https://github.com/liferay-frontend/liferay-portal/pull/354 is merged we'll have support for `propsTransformers` in `clay:button` so we could get rid of this `aui:script` and replace it with a `propsTransformer`.

Could you please guys create a task to track this so we don't forget? (I don't know your workflow). Thanks!

cc @marco-leo